### PR TITLE
Lesson 6 Task 7 - Square Root Exception

### PR DESCRIPTION
**Pull request by jean-farinea**
1. SquareRoot class created with an exception for negative numbers.

### CODE REVIEW

File: src/main/java/lesson6/task7/SquareRoot.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package lesson6.task7;
+
+public class SquareRoot {
+
+    public static void main(String[] args) {
+        try {
```
Comment (strzeleckik) [inline src/main/java/lesson6/task7/SquareRoot.java]: should be moved to unit tests

**Review by strzeleckik — CHANGES_REQUESTED** [review]
It would be good to add unit tests here

**Comment by jean-farinea** [comment]
Adjusted. Please review.

**Review by strzeleckik — APPROVED** [review]
ok

**Pull request merged**